src/ui/about.js

```diff
@@ -100,6 +103,32 @@ function About(props) {
           </React.Fragment>
         )}
 
+        <Styled.h2>Credits</Styled.h2>
+        <Styled.p>
+          Thanks to the following people and institutions for contributing to this project:
+        </Styled.p>
+        <ul>
+          <li>
+            <Styled.a href="https://github.com/slampunk">Duncan Smith</Styled.a> for starting this
+            project
+          </li>
+          <li>
+            <Styled.a href="https://github.com/cilt-uct">University of Cape Town (CILT)</Styled.a>
+            {' '}for letting Duncan start the project
+          </li>
```

Comment:
Those two sound a bit strange IMO (particularly because of the repetition). Maybe just:

```suggestion
          <li>
            <Styled.a href="https://github.com/slampunk">Duncan Smith</Styled.a> and his employer,
            {' '}<Styled.a href="https://github.com/cilt-uct">University of Cape Town (CILT)</Styled.a>,
            {' '}for starting this project
          </li>
```

(Is "his" the correct pronoun even? I don't know them)

Reply:
yes, it is

Reply:
Uhm so you'd like to keep this as two bullet points? Just want to make sure you don't accidentally marked this as "resolved".

Reply:
Yes, or at least I wouldn't put an emphasis on UCT as Duncan's employer since afaik, he started the project on his own.

```
<Styled.a href="https://github.com/slampunk">Duncan Smith</Styled.a> and the
{' '}<Styled.a href="https://github.com/cilt-uct">University of Cape Town (CILT)</Styled.a>
{' '}for starting this project
```